api/api.go

```diff
@@ -320,7 +372,7 @@ func (f *PublicController) UploadMaterial(c *gin.Context) {
 		}
 		log.Info(file)
 
-		coursematerial.CreateMaterial(f.Database, file.Name, file.Uri, 10000, id, false, nil)
+		coursematerial.CreateMaterial(f.Database, file.Name, file.Uri, f.GetIdFromCookie(c), id, false, nil)
```

Comment:
`TODO` weiter oben in der Funktion löschen

Reply:
Done